jekyll/_docs/configuration.md

```diff
@@ -203,9 +190,9 @@ machine:
 
 <h3 id="node-version">Node.js version</h3>
```

Comment:
We're using H3s after H4s. (I know you didn't do that). Plus, full Markdown is preferred here instead of HTML but might not matter soon.